docs/5.1. Transport - MQTT.md

```diff
@@ -31,7 +31,30 @@ The `broker_topic` parameter holds the MQTT topic and will always be set to the
 For a sender, this value is intended to be static. This means the IS-05 constraints should list the static value in an 'enum' for this parameter.
 For a receiver, an empty constraint object may be appropriate for this parameter.
 
-### 3.2 ext_is_07_rest_api_url
+Message types (see [Message types](2.0.%20Message%20types.md)) which use this parameter as the publishing topic:
+
+* State
+* Reboot
+* Shutdown
+
+### 3.2 connection_status_broker_topic
+
+The `connection_status_broker_topic` parameter holds the sender's MQTT connection status topic and will always be set using the following format `x-nmos/connection/{id}`.  
```

Comment:
Whilst I understand the purpose, is using 'x-nmos/connection' a little confusing given the similar namespacing for the IS-05 API? Would 'x-nmos/mqttconnection' me any better (if a little longer)?

Reply:
I agree with this.

To be honest, I have wondered all along whether the broker topic would be better as "x-nmos/events/{version}/sources/{sourceId}" and the connection status broker topic could then be something like "x-nmos/events/{version}/connections/{connectionId}". If it were down to me, I'd probably make these formats recommended rather than mandatory as well.

@mjeras, @cristian-recoseanu, what do you think?